packages/core/core-flows/src/definition/cart/workflows/complete-cart.ts

```diff
@@ -32,17 +22,25 @@ export const completeCartWorkflow = createWorkflow(
       list: false,
     })
 
+    const paymentSession = validateCartPaymentsStep({ cart })
+
+    // Question: While running cart complete, we seem to assume that the payment collection is already created.
```

Comment:
Hmm, I actually think this step was just translated from the existing cart completion strategy. I don't think we need it anymore. At least I can't think of a scenario right now.